libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -71,18 +70,27 @@ export class ButtonComponent implements AfterContentInit {
 
   ngAfterContentInit(): void {
     if (this.iconDomNode && this.iconDomNode.nativeElement) {
-      const prev = this.iconDomNode.nativeElement.previousSibling;
-      const next = this.iconDomNode.nativeElement.nextSibling;
-      if (prev && prev.nodeName !== '#comment') {
+      const iconNativeElement = this.iconDomNode.nativeElement;
```

Comment:
Consider aligning naming with similar code in repo:
```suggestion
      const iconElement = this.iconElementRef.nativeElement;
```

Reply:
Should be resolved by new commits.